.github/workflows/ci.yml

```
@@ -102,14 +102,59 @@
+
+      - name: "Download build files"
+        uses: actions/download-artifact@v4.2.1
+        with:
+          name: build-artifacts
+
+      - name: "Setup local npm registry"
+        run: ./scripts/setup-registry.sh
+
+      - name: "Install test package dependencies"
+        run: |
+          npm install 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)